extensions/extended-blocks/paid-blocks/editor.scss

```diff
@@ -40,6 +40,10 @@ $icon-background-color: #fff;
 			&:hover {
 				background: $studio-pink-30;
 			}
+			&:disabled {
+				opacity: 0.5;
+				cursor: wait;
+			}
```

Comment:
Be careful about this, maybe it requires design feedback.

Reply:
We're talking to Saul in slack

Reply:
Updated here 3e9c5e6fe48f